src/main/java/org/gatorgradle/display/CommandOutputSummary.java

```diff
@@ -198,6 +204,7 @@ public void uploadOutputSummary(List<CheckResult> failed, List<CheckResult> all)
       log.error("Failed to upload data; report endpoint specified in configuration is malformed.");
     } catch (IOException ex) {
       log.error("Exception while uploading check data: {}", ex.toString());
+      log.error("-!!!!!!!!!!!!!{}", resultListJson);
```

Comment:
This should have an actual description before the list -- the way `log.error` works is that the string is printed but the `toString` result of any following objects are placed in the corresponding `{}`. It should likely read something like `Compiled JSON to send:\n{}`. This kind of thing is very important for future users and developers who will not understand what `-!!!!!!!!!!!!!` means, even in the context of the error being printed above it; sometimes console will not always be linear (because of the parallel execution we do for some things), and so it's always good to ensure each print statement makes some sense on it's own.